crates/aptos-telemetry-service/src/log_ingest.rs

```diff
@@ -86,7 +88,12 @@ pub async fn handle_log_ingest(
     fields.insert(EPOCH_FIELD_NAME.into(), claims.epoch.to_string());
 
     let mut tags = HashMap::new();
-    tags.insert(CHAIN_ID_TAG_NAME.into(), claims.chain_id.to_string());
+    let chain_name = if claims.chain_id.id() == 3 {
+        format!("{}", claims.chain_id.id())
+    } else {
+        format!("{}", claims.chain_id)
+    };
```

Comment:
what's the distinction here?

Reply:
It's to override the name for chain_id 3. The name for chain_id 3 is `devnet` in [code](https://github.com/aptos-labs/aptos-core/blob/main/types/src/chain_id.rs#L56), but we used it for premainnet. This was done to avoid confusion. For this particular chain_id, we use the numeric value instead of the fmt::Display string.